Exercise/src/tests.cpp

```diff
@@ -0,0 +1,48 @@
+#include <gtest/gtest.h>
+
+#include "my_string.h"
+class CMyStringFixture : public ::testing::Test
+{
+protected:
+	Exercise_1::CMyString cmsMyString;
+	const size_t sTestDataLength{17};
+	const char cTestData[17]{"Some test string"};
+
+public:
+	void SetUp() override
+	{
+		cmsMyString = Exercise_1::CMyString(cTestData);
+	}
+};
+
+TEST_F(CMyStringFixture, DataCorrectnesTest) 
+{
+
+	ASSERT_EQ(sTestDataLength, cmsMyString.size());
+
+	for(std::size_t sCurrentIndex = 0; sCurrentIndex < cmsMyString.size(); sCurrentIndex++)
+	{
+		ASSERT_EQ(cTestData[sCurrentIndex], cmsMyString[sCurrentIndex]);
+	}
```

Comment:
Seems like gtest has someting like STR_EQ to compare char * no need to compare every symbol